internal/serverinstall/docker.go

```diff
@@ -291,7 +291,14 @@ func (i *DockerInstaller) InstallRunner(
 		Labels: map[string]string{
 			"waypoint-type": "runner",
 		},
-	}, nil, &network.NetworkingConfig{
+	}, &container.HostConfig{
+		// These security options are required for the runner so that
+		// Docker daemonless image building works properly.
+		SecurityOpt: []string{
+			"seccomp=unconfined",
+			"apparmor=unconfined",
+		},
+	}, &network.NetworkingConfig{
```

Comment:
Note this is only modifying how we install the _runner_ and not the _server_.

(And the runner install stuff is currently behind a hidden experimental flag)